E2E/pageObjects/game.page.ts

```diff
@@ -38,7 +44,23 @@ const locator = {
     restartGame: '//button/span[contains(text(), "Restart game")]',
     skip: '$skip-piece-btn', 
     skipButtonHighlighted: `//button[@data-testid="skip-piece-btn"][contains(@class, "selected")]`,
-    skipOtherPlayer: (username: string) =>`//*[contains(@data-testid,'${username}')]//*[@alt='skip other player button icon']`, 
+    skipOtherPlayer: (username: string) =>`//*[contains(@data-testid,'${username}')]//*[@alt='skip other player button icon']`,
+    changeProfilePicture: '//*[text() = "Change profile picture"]',
+    uploadPhoto:'//*[@data-testid = "modal-url-input-confirm-button"]',
+    confirm: '//*[@data-testid = "modal-upload-picture-button"]',
+    uploadAnotherImage: '//*[@data-testid = "modal-go-back-button"]',
```

Comment:
`data-testid` is enabled as custom locator in your plugins, so if you are searching for that exact attribute
`uploadPhoto:'$modal-go-back-button',` should be enough